.gitignore

```diff
@@ -9,5 +9,14 @@
 __pycache__/
 /authentication/migrations/
 /profiles/migrations/
+/search/migrations/
+/db.sqlite3
+/.sqlite3
+/*.sqlite3
+/forum/local_settings.py
+/node_modules
+package-lock.json
+package.json
```

Comment:
these two should not be in `.gitignore`

Reply:
Strange, because Igor said, it shoud be included in .gitignore
Also all our teammates have those files in .gitignore
Can you explain me, please, to understand everithing correct)
Thank you

Reply:
`package.json` and `package-lock.json` contain information about project dependencies. If you add any package to your project it's reflected in these files. If these two are included in `.gitignore` we can't update our dependencies and properly use `npm i` and `npm ci`.
[Check some info about package.json](https://heynode.com/tutorial/what-packagejson/)

I think Ihor means that it should NOT be included here.
cc @ProDG